pkg/pprof/pprof.go

```diff
@@ -356,7 +356,6 @@ func (c *Converter) addExecutableInfo(
 		if !(os.IsNotExist(err) || errors.Is(err, fs.ErrNotExist)) {
```

Comment:
Maybe we should revert the condition and move the else clause here?

Reply:
Definitely :D